src/components/AnnotationForm/AnnotationForm.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @flow
+ * @file Wraps a component in an IntlProvider
+ * @author Box
+ */
+
+import React from 'react';
+import noop from 'lodash/noop';
+
+import ApprovalCommentForm from '../../../third-party/components/ApprovalCommentForm';
+import Internationalize from '../Internationalize';
+import withFocus from '../withFocus';
+
+type Props = {
+    createAnnotation: Function,
```

Comment:
Would a name like `onCreate` or `onPost` make more sense/more consistent? It seems like the data model uses the idea of an Annotation for each of these comments, but conceptually they seem more like comments of an annotation rather than an annotation itself, at least for me

Reply:
Changed it to onCreate